beacon-chain/blockchain/process_block_helpers.go

```diff
@@ -332,6 +332,7 @@ func (s *Service) verifyBlkPreState(ctx context.Context, b interfaces.ReadOnlyBe
 			return errors.Wrap(err, "could not save initial sync blocks")
 		}
 		s.clearInitSyncBlocks()
+		return errors.New("could not reconstruct parent state")
```

Comment:
I dont think we can return error here, I think the caller attempt to reconstruct afterward

Reply:
oh crap we discussed offline and still this made it to the PR, sorry about that